dune-project

```diff
@@ -466,7 +466,7 @@ For more information see https://semgrep.dev
     ocolor
     ANSITerminal
     terminal_size
-    notty
+    (notty (<> :os win32))  ; todo: figure out how to placate the osemgrep_cli_interactive win32 build
```

Comment:
we can move interactive to pro (which anyway we planned to do).

Reply:
Sounds good, we can tackle that in a separate PR